src/proj_internal.h

```diff
@@ -390,6 +391,13 @@ struct PJCoordOperation {
     }
 
     ~PJCoordOperation() { proj_destroy(pj); }
+
+    bool isInstantiable();
+
+  private:
+    static constexpr int INSTANTIABLE_STATUS_UNKNOWN =
+        -1; // must be different from 0(=false) and 1(=true)
+    int isInstantiableCached = INSTANTIABLE_STATUS_UNKNOWN;
```

Comment:
Would it make sense to set it as `mutable` and avoid removing all those `const`s? A cache variable is a typical example to be mutable. (I do not know if there is any multithreading issue in this case)

Reply:
I had the same feeling when committing. I will change to mutable and restore the const's.
const is in no way a hint that somethin is multithread safe. All those objects can only be run from one PROJ_CONTEXT*